cli/cmd/plugin/ui/web/tanzu-ui/src/views/management-cluster/aws/aws-mc-common/management-credential-step/ManagementCredentials.test.tsx

```diff
@@ -51,6 +51,9 @@ describe('ManagementCredential component', () => {
         }),
         rest.get('api/provider/aws/osimages', (req, res, ctx) => {
             return res(ctx.status(200));
+        }),
+        rest.get('api/provider/aws/nodetypes', (req, res, ctx) => {
```

Comment:
the API response should contain some data.

Reply:
Yes sir. The API should contain some data. But we dont have unit test for this functionality. 
I add this API in order to pass the existed test.
As we dont use this API in the unit test, do we still need to return data?

Reply:
We have nodeInstanceTypes[0] in the initNodeProfile function above. If there is no response data , nodeInstanceTypes[0] should be undefined. Then nodeInstanceTypes[0].id would throw an error.  I'm curious why  there is no error in the unit test? Is there something that I misunderstand?

Reply:
Yes sir. You are right. But I removed the  `nodeInstanceTypes[0] ` logic. I think set the default node type is not necessary because user has not chosen the node profile (SINGLE NODE or HIGH AVAILABILITY)